ironfish/src/memPool/index.ts

```diff
@@ -3,4 +3,4 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 export * from './memPool'
 export * from './feeEstimator'
-export * from './priorityQueue'
+export { PriorityQueue } from '../utils/priorityQueue'
```

Comment:
Is it intentional that we're exporting this from a higher directory here down in the mempool index? This seems kind of strange

Reply:
Yea was just about to change this. Also added SDK change to next draft release